packages/core/src/xkeys.ts

```diff
@@ -485,11 +485,11 @@ export class XKeys extends EventEmitter {
 		} else if (this.product.backLightType === BackLightType.LINEAR) {
 			// The 40 buttons, that requires special mapping.
 
-			const ledIndex = keyIndex - 1 // 0 based linear numbering sort of...
-
-			const on: boolean = color.r > 0 || color.g > 0 || color.b > 0
+			const ledIndexBlue = keyIndex - 1 // 0 based linear numbering sort of...
+			const ledIndexRed = ledIndexBlue + this.product.backLight2offset
 
-			this._write([0, 181, ledIndex, on ? (flashing ? 2 : 1) : 0, 1])
+			this._write([0, 181, ledIndexBlue, color.b > 0 ? (flashing ? 2 : 1) : 0, 0])
+			this._write([0, 181, ledIndexRed, color.r > 0 || color.g > 0 ? (flashing ? 2 : 1) : 0, 0])
```

Comment:
Note how I also set the last bit to `0`.

From the XKE-40 data sheet:

<p><strong>8. <span class="style6">Turn On/Off Rows of Backlights</span></strong></p>
    <p>Send this output report to either turn on or off rows of the backlights. 
      Because the backlights of the XKE-40 are not laid out in rows and columns 
      this command is most useful for turning on or off all of the LEDs on a bank. 
    </p>
    
Byte 1* | Byte 2 | Byte 3 | Byte 4 | Bytes 5-36
-- | -- | -- | -- | --
Constant | Command | Bank # | State | Constant
0 | 182 | Bank | OnOff | 0


    <p><b>Bank:</b> 0 = bank 1, 1 = bank 2<br>
      <b>OnOff</b>: For all bits 0 for no backlighting, 1 for backlighting. Bit 
      0 = 1st row, bit 1=2nd row, bit 2=3rd row, bit 3=4th row, bit 4=5th row, 
      bit 5=6th row. Note the intensities are not affected by this command. Set 
      to 0 to turn off all the leds in the selected bank, set to 255 to turn them 
      all on</p>